src/common.ts

```diff
@@ -106,6 +110,7 @@ export class LoggingCommon {
   private prefix: string | undefined;
   private labels: object | undefined;
   static readonly LOGGING_TRACE_KEY = LOGGING_TRACE_KEY;
+  static readonly LOGGING_SPAN_KEY = LOGGING_SPAN_KEY;
```

Comment:
The names of the keys for traceId and spanId are not future proof (OpenTelemetry).
The value is intended to be **logging.googleapis.com/spanId** which is Cloud Logging specific.
LOGGING_TRACE_KEY is existing key. Can we create CLOUD_LOGGING_TRACE_KEY and CLOUD_LOGGING_SPAN_KEY and have developers use that instead if they want direct Cloud Trace integration rather than integration via OpenTelemetry.

Reply:
I cannot change the key names now, because it is exported as a const for users. I'll add a comment indicating that it's only for cloud logging though.